playbook/app/pb_kits/playbook/pb_advanced_table/docs/_advanced_table_with_custom_header_rails.md

```diff
@@ -0,0 +1 @@
+The optional `custom_header_renderer` item can be used within `column_definitions` to render a custom header. This example shows an Icon and Tooltip being used but other kits can be used as well.
```

Comment:
Everything looks great! Any particular reason we're naming it 'custom_header_renderer' instead of just 'header' here? Keeping parity with The react side would be ideal unless there was an issue with that name